lib/jsonrpc_client.ml

```diff
@@ -17,49 +17,95 @@
 module D = Debug.Make(struct let name = "jsonrpc_client" end)
 open D
 
-let input_json_object fin =
-	let buf = Buffer.create 1024 in
-	let brace_cnt = ref 0 in
-	let in_string = ref false in
-	let last_char () = Buffer.nth buf (Buffer.length buf - 1) in
-	let rec get () =
-		let c = input_char fin in
+exception Timeout
+exception Read_error
+
+let json_rpc_max_len = ref 65536 (* Arbitrary maximum length of RPC response *)
+let json_rpc_read_timeout = ref "60000000000"
```

Comment:
I prefer to make the configuration unit with **ms** rather than **ns** to make it user friendly.
It's really difficult for me to count the `0` and I think **ms** is precise enough for timeout.

Reply:
And why not make it `int64`s right away? Then you don't need the `Int64.of_string` for each call below.

Reply:
Why not define it as `int64`?